index.html

```diff
@@ -330,22 +335,23 @@
         (depending on the underlying platform).
       </p>
       <ul>
```

Comment:
The introduction still contains "should", and seems out of place since this is supposed to be now re-stating requirements expressed elsewhere.